lib/services/rdbmsManagement/README.md

```diff
@@ -0,0 +1,304 @@
+# Microsoft Azure SDK for Node.js - RDBMS Management
+
+This project provides a Node.js package that makes it easy to manage Microsoft Azure Storage Resources.Right now it supports:
+- **Node.js version: 6.x.x or higher**
+
+## How to Install
+
+```bash
+npm install azure-arm-rdbms
+```
+
+## How to Use
+
+### PostgreSQL
+
+#### Listing PostgreSQL servers
+
+```javascript
+const msRestAzure = require('ms-rest-azure');
+const PostgreSQLManagementClient = require('azure-arm-rdbms').PostgreSQLManagementClient;
```

Comment:
>require('azure-arm-rdbms').PostgreSQLManagementClient [](start = 35, length = 53)

Note that this is a side effect of exporting both 'postgres' and 'mysql' from the same package. Javascript programmers typically prefer to do `var postgresClient = require(packagename)` . The extra `.FooBarClient` is perceived a slight discomfort in the community, for reasons I'm not fully aware of. I would imagine that there would be no intellisense off the dot to list the available clients, but I believe the reasons for this goes beyond intellisense.

That said, it is entirely up to you to decide how you'd like to ship and maintain these two service clients - as two packages or as one :)